src/common/tests/acl_util_tests.c

```diff
@@ -1307,6 +1307,50 @@ test_ace_from_str_and_back_again(void **state)
 	check_ace_turns_back_to_same_str("UL:F:EVERYONE@:rw");
 }
 
+static void
+test_acl_from_strs_bad_input(void **state)
+{
+	struct daos_acl		*acl = NULL;
+	static const char	*valid_aces[] = {"A::OWNER@:rw"};
```

Comment:
(style)  static const char * array should probably be static const char * const

Reply:
Doesn't really fit our coding style, and feels like overkill. Planning to ignore this set of complaints unless there is objection.

Reply:
Problem is that this will require a forced landing by johann or jeff... Would be better to either appease checkpatch or figure out how to disable the warning.

Reply:
Does checkpatch require a forced landing? We've ignored this one before, though I agree it'd be better not to have the noise.

Reply:
I could be wrong about that, but I thought it did because it puts a -1 on the PR.

Reply:
I created a PR on our code review scripts to silence that warning: https://github.com/daos-stack/code_review/pull/19

Reply:
checkpatch doesn't require forced landing.   You just have to explicitly dismiss the review from daos_build